examples/wireshark.py

```diff
@@ -1,3 +1,14 @@
+"""
+Example script captures network traffic using WireShark and prints a short summary for every protocol.
+Requirements:
+  - Wireshark 2.2.5
+  - pywinauto 0.6.1+
+This example opens "Wireshark", navigates to 'network_connection_name',
+captures network traffis for 'capture_time', saves all data to temporary file,
```

Comment:
typos/grammar: `traffic for 'capture_time' seconds, saves all the data to a temporary file`